configs/anvil/config.20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil

```diff
@@ -36,37 +36,24 @@ baseDirectory = /dir/to/analysis/output
 # to be copied elsewhere (e.g. NERSC web portal)
 htmlSubdirectory = html
 
-# a list of analyses to generate.  Valid names are:
-#   'timeSeriesOHC', 'timeSeriesSST', 'climatologyMapSST',
-#   'climatologyMapSSS', 'climatologyMapMLD', 'streamfunctionMOC',
-#   'indexNino34', 'meridionalHeatTransport',
-#   'timeSeriesSeaIceAreaVol', 'climatologyMapSeaIceConcNH',
-#   'climatologyMapSeaIceConcSH', 'climatologyMapSeaIceThickNH',
-#   'climatologyMapSeaIceThickSH'
-# the following shortcuts exist:
+# a list of analyses to generate.  Valid names can be seen by running:
+#   ./run_mpas_analysis --list
```

Comment:
this should be `run_analysis`, right?

Reply:
No, this is correct.  We renamed it a long time ago.

Reply:
oh right. I guess I was confused because you changed to `./run_analysis.py` down below. I would use the same syntax here and below.

Reply:
Oh, I missed that.  The other should be `./run_mpas_analysis` and has been fixed.